src/javascript/esutil.ts

```diff
@@ -157,12 +165,24 @@ export function toScannedPolymerProperty(
   let description =
       jsdoc.removeLeadingAsterisks(getAttachedComment(node) || '').trim();
 
+  const name = objectKeyToString(node.key);
+  if (name == null) {
```

Comment:
Oh right, I remember we've talked about this before. Using `(name == null)` to check for falseyness is deceptive to me as a reviewer, because it feels like the intention is to check something more specific about the variable & `null` and maybe the `==` was a mistake.  `(!name)` more explicitly checks for falseyness.

Can we switch this to `(!name)` here and going forward?